lib/fakeroot.py

```diff
@@ -221,6 +224,24 @@
      "cmds": ["apt", "apt-get", "dpkg"],
      "each": ["fakeroot"] },
 
+   # Fedora notes:
+   #
+   # 1. The supported versions were chosen somewhat arbitrarily based on the
+   #    release versions available for building (what was on Dockerhub). To
+   #    extend support to additional versions one need only adjust the match
+   #    config.
+   #
+   # 2. The fakeroot package is in the base repository set so enabling EPEL is
+   #    not required.
+
+   "fedora":
+   { "name": "Fedora 24-35,",
+     "match":  ("/etc/fedora-release", r"release (2[4-9]|3[0-5])"),
```

Comment:
I think Dave Love had a good point. Can we remove the high end of this range test?

Reply:
Can do, I'll just check that it is above version 24.